TodoList.Web/Controllers/TodosController.cs

```diff
@@ -34,15 +34,23 @@ public async Task<IActionResult> Home()
             var currentUser = await _userManager.GetUserAsync(User);
             if (currentUser == null) return Challenge();
 
+            var now = DateTime.Now;
+            var cal = new CalendarViewModel(now.Month, now.Year);
```

Comment:
Please give it a better name such as `calendarModel`

Reply:
Definitely, will do.